static/app/views/monitors/components/overviewTimeline/timelineTableRow.tsx

```diff
@@ -225,6 +230,10 @@ const Name = styled('h3')`
   word-break: break-word;
 `;
 
+const StyledIconUnsubscribed = styled(IconUnsubscribed)`
```

Comment:
Can you just change `Name` to be flex w/ a gap instead of restyling the icon?

Reply:
hmm tried a couple things out and might need some help, so initially due to the icon sizing adding flex looks something like:
<img width="361" alt="image" src="https://github.com/getsentry/sentry/assets/9372512/d4de42da-f895-435a-bf88-71aa585525f2">
which we could fix with `align-items: center` but yields awkward behavior when the cron title wraps
<img width="363" alt="image" src="https://github.com/getsentry/sentry/assets/9372512/aa779c0c-f085-463b-a51a-5ab86d7476be">
of course going to what i have now just ends up looking like
<img width="372" alt="image" src="https://github.com/getsentry/sentry/assets/9372512/abc93236-ea7f-4a01-8e33-0f0e13bca497">

perhaps its best to just place the mute icon directly to the left of the disabled icon